updating/update-using-custom-machine-config-pools.adoc

```diff
@@ -62,12 +127,16 @@ include::modules/update-using-custom-machine-config-pools-pause.adoc[leveloffset
 [id="update-using-custom-machine-config-pools-update_{context}"]
 == Performing the cluster update
 
+<<<<<<< HEAD:updating/update-using-custom-machine-config-pools.adoc
 When the MCPs enter ready state, you can peform the cluster update. See one of the following update methods, as appropriate for your cluster:
+=======
+After the machine config pools (MCP) enter a ready state, you can perform the cluster update. See one of the following update methods, as appropriate for your cluster:
+>>>>>>> 1301ec6707 (OSDOCS-9497: editing canary rollout update page):updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc
```

Comment:
Waiting for a forced push to remove these changes that I forgot to include because I didn't save the file.